packages/visibility_detector/pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: visibility_detector
-version: 0.4.0+2
+version: 0.4.0+3
```

Comment:
I think this needs to be a bigger bump right?

Reply:
Why's that? All changes are backwards compatible.